datalake/src/test/java/com/sequenceiq/datalake/service/upgrade/OrderedOSUpgradeRequestProviderTest.java

```diff
@@ -56,19 +62,53 @@ void testCreateOrderedOSUpgradeSetRequest() {
                 createInstanceMetadata(GATEWAY, 1)
         )));
         instanceGroups.add(createInstanceGroup(Set.of(createInstanceMetadata(AUXILIARY, 0))));
+        instanceGroups.add(createInstanceGroup(Set.of(createInstanceMetadata(SOLRHG, 3))));
```

Comment:
please create separate unit test for Enterprise DL test

Reply:
will change.